styles/base/_links.scss

```diff
@@ -72,20 +72,27 @@
 
   &:hover,
   &:active,
-  &:focus,
-  &:visited {
+  &:focus {
     border-bottom: $sprk-link-has-icon-hover-border-bottom;
     color: $sprk-link-has-icon-hover-color-text;
   }
+
+  &:visited {
+    border-bottom: $sprk-link-has-icon-visited-border-bottom;
+    color: $sprk-link-has-icon-visited-color-text;
+  }
 }
 
 .sprk-b-Link--has-icon .sprk-c-Icon {
   color: $sprk-link-has-icon-color-icon;
   fill: $sprk-link-has-icon-fill;
   stroke: $sprk-link-has-icon-stroke;
+  transition: $sprk-link-transition;
```

Comment:
Added this so the icon and text would animate together instead of changing color at different times.